src/components/views/UserPage.js

```diff
@@ -153,6 +158,35 @@ const UserPage = () =>{
         )
     }
 
+    // MAP
+    let map = null;
+    let position = [0, 0];  // default position
+    let zoomScale = 14;     // set zoom
+    if(user){
+        position = [user.latitude, user.longitude]     // latitude, longitude
+        if(position[0]===0){
```

Comment:
shouldn't this be tested in both directions? Yes it is highly unlikely that somone manages to be exactly on the equator or exactly on the greenwich line but still..